xcoll/colldb.py

```diff
@@ -217,20 +232,32 @@ def from_SixTrack(cls, file, **kwargs):
 
         df = pd.read_csv(io.StringIO(coll_data_string), delim_whitespace=True,
                         index_col=False, names=names)
-
+        #print('\n family_types: ', family_types)
+        #print('\n family_settings: ', family_settings, '\n')
         df.insert(5,'stage', df['gap'].apply(lambda s: family_types.get(s, 'UNKNOWN')))
         sides = df['name'].apply(lambda s: side.get(s, 0))
+        crystals = df['name'].apply(lambda s: crystal.get(s, 0))
+        #df['family'] = df['gap']
         df['gap'] = df['gap'].apply(lambda s: float(family_settings.get(s, s)))
         df['name'] = df['name'].str.lower() # Make the names lowercase for easy processing
         df['parking'] = 0.025
+        df['bend'] = df['name'].apply(lambda s: bend.get(s, 0))
```

Comment:
This should default to `None` otherwise the tests fail:

`df['bend'] = df['name'].apply(lambda s: bend.get(s, None))`